src/windows/BarcodeScannerProxy.js

```diff
@@ -339,11 +350,31 @@ module.exports = {
             }
 
             focusControl.configure(focusConfig);
-            // Need to wrap this in setTimeout since continuous focus should start only after preview has started. See
-            // 'Remarks' at https://msdn.microsoft.com/en-us/library/windows/apps/windows.media.devices.focuscontrol.configure.aspx
-            return WinJS.Promise.timeout(200)
-            .then(function () {
-                return focusControl.focusAsync();
+
+            // Continuous focus should start only after preview has started. See 'Remarks' at 
+            // https://msdn.microsoft.com/en-us/library/windows/apps/windows.media.devices.focuscontrol.configure.aspx
+            return new WinJS.Promise(function (complete, error, progress) {
+                function onplaying() {
+                    capturePreview.removeEventListener("playing", onplaying, false);
+
+                    try {
+                        focusControl.focusAsync().done(function () {
+                            complete();
+                        }, function (e) {
+                            error(e);
+                        });
+                    } catch (e) {
+                        console.error(e);
+                        // This happens on mutliple taps
+                        if (e.number !== OPERATION_IS_IN_PROGRESS) {
+                            console.error('focusAsync failed: ' + e);
+                            error(e);
+                        }
+                        complete();
+                    }
+                }
+
+                capturePreview.addEventListener("playing", onplaying, false);
```

Comment:
Can we check for `capturePreview.palying` property before attaching a handler? If it is `true` already we could call `focusAsync` immediately